scripts/run_predicators_baseline_experiments.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+START_SEED=456
+NUM_SEEDS=10
+FILE="scripts/submit.py"
+NUM_TRAIN_TASKS="200"
+ALL_ENVS=(
+    "cover"
+    "pybullet_blocks"
+    "painting"
+    "tools"
+)
+
+for SEED in $(seq $START_SEED $((NUM_SEEDS+START_SEED-1))); do
+    for ENV in ${ALL_ENVS[@]}; do
+        # Main approach.
+        python $FILE --experiment_id ${ENV}_main_${NUM_TRAIN_TASKS}demo --env $ENV --approach grammar_search_invention --excluded_predicates all --seed $SEED --num_train_tasks $NUM_TRAIN_TASKS
+
+        # Ablations.
+        # Note: downrefeval is main but with --sesame_max_skeletons_optimized 1 during evaluation only. We can only run this using `--load_approach` since we don't allow grammar_search_expected_nodes_max_skeletons to be greater than sesame_max_skeletons_optimized during invention.
+        python $FILE --experiment_id ${ENV}_downrefscore_${NUM_TRAIN_TASKS}demo --env $ENV --approach grammar_search_invention --grammar_search_expected_nodes_max_skeletons 1 --excluded_predicates all --seed $SEED --num_train_tasks $NUM_TRAIN_TASKS
+        python $FILE --experiment_id ${ENV}_noinventallexclude_${NUM_TRAIN_TASKS}demo --env $ENV --approach nsrt_learning --excluded_predicates all --seed $SEED --num_train_tasks $NUM_TRAIN_TASKS
+        python $FILE --experiment_id ${ENV}_noinventnoexclude_${NUM_TRAIN_TASKS}demo --env $ENV --approach nsrt_learning --seed $SEED --num_train_tasks $NUM_TRAIN_TASKS
```

Comment:
let's rename all experiment IDs to match paper? it took me a while to remember that `noinventnoexclude` is Manual, etc

Reply:
nah